source/developer/dev-setup-windows.rst

```diff
@@ -14,12 +14,18 @@ Installing Developer Components on Windows
 3. Download and install Node.js from https://nodejs.org/.
 4. Download and install Go 1.8 from https://golang.org/dl/.
 5. Install Yarn. Go to https://yarnpkg.com/en/docs/install#windows-tab and follow the installation instructions.
-6. Fork Mattermost on GitHub.com from https://github.com/mattermost/platform, then:
+6. Fork Mattermost server and webapp on GitHub from https://github.com/mattermost/mattermost-server and https://github.com/mattermost/mattermost-server, then:
```

Comment:
Shouldn't this read "Fork Mattermost server and webapp on GitHub from https://github.com/mattermost/mattermost-server and https://github.com/mattermost/mattermost-webapp, then:"